src/screens/PasswordRecovery/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import EmailForm from './componentes/EmailForm'
+import { isValidEmail } from '../authentication/utils'
+import * as firebase from 'firebase'
+import { View, KeyboardAvoidingView } from 'react-native'
+import { SvgXml } from 'react-native-svg'
+import { Title, Description } from '~/components/texts'
+import styles from './styles'
+import Image from './resources/recover.svg'
+import GoBackToSignIn from './componentes/GoBackToSignIn'
+import NavigationService from '~/navigationService'
+
+const PasswordRecovery = () => {
+  const [error, setError] = useState(null)
+  let emailSent = false
+
+  const handleSubmit = async (values) => {
+    setError(null)
+    let email = null
+    var auth = firebase.auth()
+    if (isValidEmail(values.email)) {
+      email = values.email
+      auth.sendPasswordResetEmail(email).then(function () {
+        emailSent = true
+        NavigationService.navigate('Login', { emailSent })
+      }).catch(function (error) {
+        setError(error)
+      })
+    } else {
+      setError(true)
+    }
+  }
```

Comment:
Please consider moving this authentication logic to `authenticationService`, as it is better for separation of concerns. Also use `let` or `const` instead of `var`.